jekyll/_cci2/optimizations.md

```diff
@@ -27,109 +27,48 @@ Choosing the right docker image for your project can have huge impact on build t
-      - run: <command>
-workflows: # Here we can orchestrate our jobs into a workflow
-  version: 2
-  build_and_test: # A single workflow named "build_and_test"
-    jobs: # we run our `build` job and `test` job concurrently.
-      - build
-      - test
-```
-
-You can view more examples of workflows in the  [CircleCI demo workflows repo](https://github.com/CircleCI-Public/circleci-demo-workflows/).
+Workflows provide a means to define a collection of jobs and their run order. If at any point in your build you see a step where two jobs could happily run independent of one another, workflows may be helpful. Workflows also provide several other features to augment and improve your build configuration. Read more about workflows on the [Workflow]({{site.baseurl}}/2.0/workflows/) page.
+
+* You can view examples of workflows in the  [CircleCI demo workflows repo](https://github.com/CircleCI-Public/circleci-demo-workflows/).
 
 ## Workspaces
 {: #workspaces }
 
-**Note**: Using workspaces presumes that you are also using [workflows](#workflows).
+**Note**: Using workspaces presumes that you are also using [Workflows](#workflows).
 
 Workspaces are used to pass along data that is _unique to a run_ and is needed for _downstream jobs_. So, if you are using workflows, a job run earlier in your build might fetch data and then make it _available later_ for jobs that run later in a build.
 
-To persist data from a job and make it available to downstream jobs via the [`attach_workspace`]({{ site.baseurl}}/2.0/configuration-reference#attachworkspace) key, configure the job to use the [`persist_to_workspace`]({{ site.baseurl}}/2.0/configuration-reference#persisttoworkspace) key. Files and directories named in the paths: property of `persist_to_workspace` will be uploaded to the workflow’s temporary workspace relative to the directory specified with the root key. The files and directories are then uploaded and made available for subsequent jobs (and re-runs of the workflow) to use.
+To persist data from a job and make it available to downstream jobs via the [`attach_workspace`]({{site.baseurl}}/2.0/configuration-reference#attachworkspace) key, configure the job to use the [`persist_to_workspace`]({{site.baseurl}}/2.0/configuration-reference#persisttoworkspace) key. Files and directories named in the paths: property of `persist_to_workspace` will be uploaded to the workflow’s temporary workspace relative to the directory specified with the root key. The files and directories are then uploaded and made available for subsequent jobs (and re-runs of the workflow) to use.
+
+* Read more on the [Workspaces]({{site.baseurl}}/2.0/workspaces/) page.
 
-Read more about how to use workspaces in the [workflows document]({{site.baseurl}}/2.0/workflows/#using-workspaces-to-share-data-among-jobs).
+* The [Workflows]({{site.baseurl}}/2.0/workflows/#using-workspaces-to-share-data-among-jobs) page has additional information about using workspaces to share data among jobs.
```

Comment:
This line can go, I moved it all to the Worspaces page I think, linked to in the line above.